CONTRIBUTING.md

```diff
@@ -65,19 +65,52 @@ Thus:
 - if the list includes `next` but no release tags: the fix/enhancement will come in the nearest minor release.
 - if the list includes `main` but no release tags: the fix/enhancement will come in the nearest patch release.
 
+## Proposing new changes
+
+If you want to add functionality and features that don't fall under an existing and accepted issue please add a [Kubernetes Enhancement Proposal (KEP)][keps] in a pull request to the `railgun/keps/` directory.
+
+To the best of your ability: take some time before you propose something new to search through some of the existing KEPs to make sure that your proposal isn't a duplicate nor is it colliding with anything others.
+
+Read our [KEP Documentation][kepstart] which provides some guidelines about how to get started and contributing to new or existing KEPs.
+
+[keps]:https://github.com/kubernetes/enhancements/blob/master/keps/README.md
+[kepstart]:https://github.com/Kong/kubernetes-ingress-controller/blob/next/railgun/keps/README.md
+
 ## Development environment
 
 ## Environment
 
 - Golang version matching our [`Dockerfile`](./Dockerfile) installed
-- Access to a k8s cluster, you can use Minikube or GKE
-- make
-- Docker (for building)
+- [Kubebuilder][kubebuilder] `v3.1.x`+
+- [GNU Make][make] `v4.x`+
+- [Docker][docker] (for building) `v20.10.x`+
```

Comment:
These version numbers will rot over time.

Can we, instead of making claims about version numbers, point to places in the codebase where the version used by our automation is defined authoritatively?

Reply:
changed in 17aeaef30c13f43132971a37a257bac9717f3342, but just changed back to what was here previously.